src/components/sidebar/right-sidebar/RightSidebar.tsx

```diff
@@ -1,8 +1,6 @@
 import React from "react";
 import "../sidebar.scss";
-import Datepicker from "./datepicker/Datepicker";

```

Comment:
is the datepicker not a part of the right sidebar?

Reply:
It is, but I believe this was unused code from a different implementation